AccountService.js

```diff
@@ -6,14 +6,15 @@
 import axios from 'axios';
 
 const headers = {Accept: 'application/ld+json, application/json'};
-/**
- * This service is used to encapsulate all account
- * based activity for a project.
- * @param {Object} [config = {urls: {base: '/accounts'}}]
- * @param {Object} [config.urls = {}]
- * @param {string} [config.urls.base = /accounts]
- */
+
 export class AccountService {
+  /**
+   * This service is used to encapsulate all account
+   * based activity for a project.
+   *
+   * @param {object} [config = {urls: {base: '/accounts'}}] -
```

Comment:
Not sure what descriptions to put here. L15-16

Reply:
"The urls to use." will be fine.